hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -491,9 +502,8 @@ export class Om extends React.Component<Record<string, object>, IOmdbInsightsSta
     cancelMismatchedEndpointSignal = controller;
     request.then(mismatchContainersResponse => {
       const mismatchContainers: IContainerResponse[] = mismatchContainersResponse && mismatchContainersResponse.data && mismatchContainersResponse.data.containerDiscrepancyInfo;
-      if (mismatchContainersResponse && mismatchContainersResponse.data && mismatchContainersResponse.data.lastKey === null) {
+      if (mismatchContainers && mismatchContainers.length === 0) {
         //No Further Records may be last record
-        mismatchPrevKeyList = [0];
```

Comment:
Shouldn't we do a `prevKeyListMap.clear();` here?

Reply:
@dombizita 
If we clear map then we could go back to previous pages. As per New Changes from Last Pages we need to go back to previous pages.